### PR TITLE
Bump operator-sdk to version 1.2.0

### PR DESCRIPTION
**Pull request by jweibel22**
migrated by scaffolding a new project with operator-sdk init and copying in code from old project

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
+FROM golang:1.13 as builder
```
Comment (Crevil) [inline Dockerfile:2]: Shouldn't we use the latest Go version here?

```suggestion
FROM golang:1.15.7 as builder
```

File: PROJECT [hunk 2]
```diff
@@ -0,0 +1,11 @@
+domain: lunar.tech
+layout: go.kubebuilder.io/v2
+projectName: hubble-rbac-controller-new
```
Comment (Crevil) [inline PROJECT:3]: Can we drop the `-new` suffix?

**Review by Crevil — APPROVED** [review]
LGTM

File: PROJECT (hunk 2 above)
Comment (jweibel22) [reply PROJECT:3]: oops :-)

**Pull request merged**